server/lib/geolocation.js

```diff
@@ -0,0 +1,13 @@
+const request = require('superagent');
+
+const getGeoLocation = (postcode, callback) => {
```

Comment:
Are there any tests for this? Would be good to have some checks to make sure it works.